pkg/iris/scan.go

```diff
@@ -134,11 +121,12 @@ func (s *scanner) Scan(getKubernetesAPI kubernetes.GetKubernetesAPI, config *res
 	return err
 }
 
-func (s scanner) ScanNamespace(k8sApi *kubernetes.API, mapper Mapper, namespaces []corev1.Namespace, cluster ClusterDTO, workspaceId string) ([]models.DiscoveryItem, error) {
+func (s scanner) ScanNamespace(k8sApi *kubernetes.API, mapper Mapper, namespaces []corev1.Namespace, cluster ClusterDTO, workspaceId string, config kubernetesConfig) (interface{}, error) {
```

Comment:
Had you thought about creating a supertype "Event" that contains both the discovery and command events? Maybe not a big deal, but having a generic interface here doesn't provide to much type safety I guess.

Reply:
True, but since there are only two common fields between them, I didn't bother with the super type to make it more straightforward